live-examples/html-examples/input/checkbox.html

```diff
@@ -0,0 +1,15 @@
+<fieldset>
+    <legend>Choose some monster features</legend>
+
+    <input type="checkbox"
+           id="scales" name="feature" value="scales" checked>
```

Comment:
Nit: Code style: Even though it is not required with HTML5, I still feel it is good practice to close self closing tags:

```
<input type="checkbox"
           id="scales" name="feature" value="scales" checked />
```